src/printer.ts

```diff
@@ -838,6 +854,24 @@ export class PugPrinter {
 			this.result += this.indentString.repeat(this.indentLevel);
 		}
 		this.wrapAttributes = false;
+
+		if (this.classLiteralToAttribute.length) {
+			if (this.previousToken?.type === 'start-attributes') {
+				this.result += '(';
+			}
+
+			if (this.previousToken?.type === 'attribute') {
```

Comment:
**suggestion (performance):** Using at least an `else if` here could safe one check, cause when it was already a `start-attribute` it cannot be an `attribute` anymore